tor/core/inbox.py

```diff
@@ -23,6 +22,16 @@
 ]
 
 
+def send_reply_to_slack(item, config):
+    send_to_slack(
+        'Unknown reply by **{author}**, {subject}: {body}'.format(
+            author=item.author,
+            body=item.body,
+            subject=item.subject,
+        ), config
+    )
```

Comment:
We might also want to mark this item as read so we don't keep forwarding the message to slack every time the bot goes through the inbox. I might be missing something here, though. Thoughts, @itsthejoker?

Reply:
Nah, I wanted to spam you guys with messages. /s